asgi_webdav/auth.py

```diff
@@ -6,6 +6,7 @@
 from enum import IntEnum
 from logging import getLogger
 from uuid import uuid4
+from typing import Optional, Tuple
```

Comment:
Don't this:`from typing import Optional, Tuple`. This project minimum Python version requirement is v3.10

Reply:
I see, I can resubmit the PR without the typing hints changes.

I made the change as I get  an error from pyright that tuple expressions are not allowed:
For example `(DAVUser | None, str)`

Without `typing` package I can make pyright happy with the built-in `tuple[DAVUser | None, str]`. Would that work for you?